python.html.markdown

```diff
@@ -144,6 +145,10 @@ bool("")  # => False
 # Python has a print statement
 print "I'm Python. Nice to meet you!"
 
+# Simple way to get input data from console
+input_string_var = raw_input("Enter some data: ") # Data is stored as a string
+input_number_var = input("Enter a number: ") # Data is stored as a number
```

Comment:
Actually, data is not stored as a number all the time. If I enter "learn x in y" for the input prompt, it will be a string, not an int. So it *can* be an int but also *can* be something else.